docs/developer/getting-started.md

```diff
@@ -0,0 +1,161 @@
+
+:::tip
+If you want to develop a Graasp app, you don't need the full setup.
+You can head over to [the Graasp App Development section](/developer/apps/intro).
+:::
+
+:::important
+We will assume that you have a UNIX compatible machine at your disposal (Mac, Linux or Windows with WSL). All instructions should work independently of the environment. If you encounter problems, please [open an issue](https://github.com/graasp/docs/issues/new).
+:::
+
+## Requirements
+
+If not done already please install these required tools
+
+- [Docker Desktop](https://www.docker.com/products/docker-desktop/) (if using Mac or Windows), on Linux you should be good with installing the docker package.
+- [Node](https://nodejs.org/en/download) version 20 or above with `nvm` (recommend)
+- `git` for version control operations
+
+### Enabling Yarn with `nvm`
+
+Yarn is a package manager for node and javascript projects. It is faster than `npm`.
+The recommended way of enabling yarn is using `corepack`.
+
+Start by running `corepack enable`
+Then `corepack prepare yarn@stable --activate`
+
+You should now have `yarn` enabled with your version of node defined by nvm. To check run `yarn --version` or `which yarn`
+
+:::tip
+This operation should be repeated every time you update the version of node you use.
+:::
+
+## Backend installation
+
+### Getting the source code
+
+Go to the [Graasp backend repo](https://github.com/graasp/graasp) and clone it locally.
+
+### Opening dev containers
+
+Open the repo in VSCode. Make sure the ["Dev Containers" extension](https://code.visualstudio.com/docs/devcontainers/tutorial#_install-the-extension) is downloaded and enabled.
+
+Then open the folder in the dev-container by using the command palette `cmd`+`shift`+`P` (or `ctrl` instead of `cmd`), and typing **Open Folder in Container**.
+
+### Installing dependencies
+
+After the containers are up and running execute `yarn` (you can do `yarn install` which is the same) in the container terminal to install dependencies.
+
+### Add environment variables
+
+Go to [the configuration in the Readme](https://github.com/graasp/graasp#configuration) and copy it to a new root file in your backend folder named `.env.development`.
+
+You will need to generate a few secret values. Open the `.env.development` file and look of places that ask for these.
+The first one should be `SECURE_SESSION_SECRET_KEY`.
+To generate a secret you can simply run:
+
+```sh
+npx @fastify/secure-session > secret-key && node -e "let fs=require('fs'),file=path.join(__dirname, 'secret-key');console.log(fs.readFileSync(file).toString('hex'));fs.unlinkSync(file)"
+```
+
+Replace the `<content>` placeholder with the output of the command (a string of numbers).
+Repeat the process until you have generated all necessary keys. You can search for `<content>` in the file in order to located them.
+
+### Running backend
+
+We should be all set. Run `yarn watch` to start the backend server.
+
```

Comment:
Maybe have a troubleshoot section (here or below). For example the backend couldn't get started because some of the users had a postgres container already running on their machine, so they should check if localhost:port is already taken or not.

Reply:
There is a specific [troubleshooting page](https://github.com/graasp/docs/blob/main/docs/developer/trouble-shooting.md) 
Could you add something describing this issue there ? I am not very familiar with it, as I never encountered it.